src/main/plugin/iso19139.nap/formatter/ec-view/view.xsl

```diff
@@ -393,7 +393,7 @@
           <xsl:variable name="geom" select="concat('POLYGON((', $minx, ' ', $miny,',',$maxx,' ',$miny,',',$maxx,' ',$maxy,',',$minx,' ',$maxy,',',$minx,' ',$miny, '))')"/>
 
           <xsl:if test="$hasGeometry">
-            <span class="bold"><xsl:value-of select="$schemaStrings/GeographicExtent"/></span><br/>
+            <span class="h6"><xsl:value-of select="$schemaStrings/GeographicExtent"/></span><br/>
```

Comment:
This looks good, thanks @bo-lu

Reply:
@jodygarnett 
Should this change be applied to 3.10.x as well?